baseplate/lib/edge_context.py

```diff
@@ -353,11 +365,13 @@ class EdgeRequestContext:
 
     _HEADER_PROTOCOL_FACTORY = TBinaryProtocolAcceleratedFactory()
 
-    def __init__(self, authn_token_validator, header):
+    def __init__(
+        self, authn_token_validator: AuthenticationTokenValidator, header: Optional[bytes]
+    ):
         self._authn_token_validator = authn_token_validator
         self._header = header
 
-    def attach_context(self, context):
+    def attach_context(self, context: Any) -> None:
```

Comment:
Should we just go ahead and type this to `RequestContext`?

Reply:
done, thanks